spec/install/gemfile/git_spec.rb

```diff
@@ -82,7 +82,23 @@
         gem "foo", "1.1", :git => "#{lib_path("foo-1.0")}"
       G
 
-      expect(out).to include("Source contains 'foo' at: 1.0")
+      expect(out).to include("Source contains 'foo' at: 1.0 ruby")
+    end
+
+    it "complains with platform (if not ruby) if pinned specs don't exist in the git repo" do
```

Comment:
Basically, not sure how to make the `Gemfile` something like this:
```
        platforms :jruby do
          gem "only_java", "1.2", :git => "#{lib_path("only_java-1.0-java")}"
        end
```
and have it detect both 1.0 and 1.1 versions of `only_java` as it seems to be going by the `only_java.gemspec` specification in the `tmp/libs/only_java-1.0-java` directory. In this gemspec, only one version can be specified.